Makefile

```diff
@@ -29,8 +29,8 @@ APPNAME      = "Cardano ADA"
 
 # Application version
 APPVERSION_M = 7
-APPVERSION_N = 2
-APPVERSION_P = 0
+APPVERSION_N = 1
```

Comment:
Downgrade version ?

Reply:
Yes, to keep compatibility with client apps. It was asked by @vforgeoux-ledger.
I increment the patch instead which will not break things.